apps/sales/urls.py

```diff
@@ -18,8 +18,7 @@
 router.register(r'order_shipments', OrderShipmentsView)
 
 urlpatterns = [
-    path('',include(router.urls))
-]
-
-
-
+    path('',include(router.urls)),
+    path('sales-order-operation/', SaleOrderOneView.as_view(), name='sales-order-operation-list-create'),
+    path('sales-order-operation/<str:pk>/', SaleOrderOneView.as_view(), name='sales-order-operation-detail-update-delete'),
```

Comment:
Change the above sales-order-operation to sale_order and change the name attribute accordingly

Reply:
updated